python/acquire-zarr-py.cpp

```diff
@@ -588,6 +599,12 @@ PYBIND11_MODULE(acquire_zarr, m)
       .value(dimension_type_to_str(ZarrDimensionType_Other),
              ZarrDimensionType_Other);
 
+    py::enum_<ZarrDownsamplingMethod>(m, "DownsamplingMethod")
```

Comment:
Can users set this by string? If so, is the string recognition case independent?